### DISCUSSION TITLE
Which font should I use?

### DISCUSSION THREAD

**Discussion by TheMowgliMan** [TheMowgliMan/Country-Clash#8] [Polls]
I've been browsing Google Fonts for a default font for CCTG. Pick your favorite, or choose mention a different one you'd like. This should continue for a couple weeks for new employees to choose.

**Comment by 3quarterscolon** [comment]
How about Playfair Display or Macondo for title text or whatever, and something like [this font](https://fonts.google.com/specimen/BenchNine) for the bulk of the text?

**Reply by TheMowgliMan** [reply]
That seems good. I'll remember it.

**Reply by TheMowgliMan** [reply]
> How about Playfair Display or Macondo for title text or whatever, and something like [this font](https://fonts.google.com/specimen/BenchNine) for the bulk of the text?

I think Nobile is a better choice, as the squished appearance of Bench Nine is hard to read on my screen.

**Reply by TheMowgliMan** [reply]
I'll add Nobile as the new default font, but add Macondo back to the main menu buttons.
